### ISSUE TITLE
UMS aborts abruptly

### ISSUE THREAD

**Issue by sherebiahtishbi** [UniversalMediaServer/UniversalMediaServer#4859]
I run UMS 14.2.1 on Fedora 40. After upgrading from previous UMS version it ran seamless for the first time. But now it gives error aborting UMS abruptly as shown below.

```
WARN  08:08:23.012 [Media Scanner] Unhandled exception while resolving IMG_4228: Cannot invoke "net.pms.encoders.Engine.isActive()" because the return value of "net.pms.encoders.EngineFactory.getActiveEngine(net.pms.encoders.EngineId)" is null
double free or corruption (out)
Aborted (core dumped)

```

There is nothing come up in google search as well. Let me know what all other information you need to troubleshoot and I will provide as much as I can.

**Comment by SubJunk** [comment]
@sherebiahtishbi thanks for reporting this, please follow the instructions at https://github.com/UniversalMediaServer/UniversalMediaServer/issues/new to generate a debugging zip file and I will help

**Comment by sherebiahtishbi** [comment]
Thanks for the response @SubJunk . Attached is the trace log 

[trace-07302024.zip](https://github.com/user-attachments/files/16428376/trace-07302024.zip)

**Comment by SubJunk** [comment]
@sherebiahtishbi this fix is merged and being released now, would you like me to upload a build for you?

**Comment by sherebiahtishbi** [comment]
Yes that would be great! Much appreciated.

**Comment by sherebiahtishbi** [comment]
Also a config setting to exclude certain file extensions would be a desirable feature. I am trying to add that in my local copy in the parser class but will take time. Meanwhile someone who is more famiiar wirh code should be able to do it in no time i believe.

**Comment by SubJunk** [comment]
@sherebiahtishbi the fix has been released in 14.4.0 but in case you don't have access to that, I also uploaded this for you https://www.universalmediaserver.com/uploads/UMS-sherebiahtishbi-4859.exe
Please let me know when you have downloaded it so I can remove it
